_pages/en_US/godmode9-usage.txt

```diff
@@ -122,18 +122,52 @@ Insert the game cartridge you intend to dump into your device
   + **NDS Game Cartridge:** Press (A) on `[TitleID].trim.nds` to select it, then select "Copy to 0:/gm9/out"
 1. Your installable `.cia` or non-installable `.nds` formatted file will be outputted to the `/gm9/out/` folder on your SD card
 
+## Dumping a Game Cartridges romfs/exefs
+
+1. Insert a game cartrige you wish to dump from
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Hover over "GAMECART ()"
+1. Press (A), wait (could take up to 20 seconds)
+1. Hover over the bigest file ending with `.3ds` and press (A)
+1. Select "NCCH image options"
+1. Select "Mount image to drive"
+1. Press (A) to continue
+1. Hover over `exefs.bin`, hold (R) and press (A) to open "directory options", select "copy to 0:/gm9/out", do the same with `romfs.bin`
```

Comment:
Honestly a bit unclear what you're trying to do here 'cus all this does is copy the `romfs.bin` and `exefs.bin` files which aren't that useful on their own. The main purpose for this is likely game modding in which case you should unpack the romfs which can be done in GM9